docs/types/reference-types.rst

```diff
@@ -130,10 +126,45 @@ restrictions for types apply, in that mappings can only be stored in the
 It is possible to mark state variable arrays ``public`` and have Solidity create a :ref:`getter <visibility-and-getters>`.
 The numeric index becomes a required parameter for the getter.
 
+.. _array_indexing:
+
+Array Indexing
+--------------
```

Comment:
This puts it on the same level as arrays, doesn't it?

Reply:
Not sure. I'm not super familiar with rst, so I sometimes make educated guesses. Please keep that in mind when you're looking at my suggested edits.

Reply:
If this was meant to be a subsection, it should look like this (because `^` is the character already used above in the document for this nesting level):

```suggestion
^^^^^^^^^^^^^^
```